### ISSUE TITLE
Clearing a to-many relationship in a "normal" patch

### ISSUE THREAD

**Issue by TristanKobalt** [wopian/kitsu#517]
I am trying to update a value on a resource, and clear a to-many relationship at the same time like so:

`            
api.update('my-resource', {
    id: 1,
    my-field: 'new value',
    my-to-many-relationship: [] 
});
`

However, the `my-to-many-relationship` is not being recognized as a relationship in this case. Am I doing something wrong? I would like this operation to remain atomical.

Thanks in advance!

**Comment by wopian** [comment]
The syntax should be:

```js
api.update('my-resource', { 
  id: 1, 
  my_field: 'new value', 
  my_to_many_relationship: { data: [] } 
});
```

https://github.com/wopian/kitsu/blob/master/packages/kitsu/MIGRATING.md#link-objects

**Comment by wopian** [comment]
Fixed in [`10.0.0-alpha.10`](https://github.com/wopian/kitsu/releases/tag/v10.0.0-alpha.10). 

Note that in `10.x` ([see Migration Guide#link-objects](https://github.com/wopian/kitsu/blob/master/packages/kitsu/MIGRATING.md#link-objects)), relationships are now always objects with a `data` object/array.

```js
api.update('my-resource', { 
  id: 1, 
  my_field: 'new value', 
  my_to_many_relationship: { data: [] },
  my_to_one_relationship: { data: null }
});
```
